py/static/js/login-checker.js

```diff
@@ -0,0 +1,35 @@
+//uncomment this line and comment the next line if you decided to put the js to login page
+//document.addEventListener("DOMContentLoaded", function (evt) {
```

Comment:
No need to leave comments which you don't need :)

**Thanks for speedy fix of PR!** I'm rushing to work now, but I hope I'll be able to check and merge this soon and then I'll update the site!